transformations/question_to_statement/transformation.py

```diff
@@ -0,0 +1,121 @@
+from typing import Tuple, List
+import nltk
+from pyinflect import getInflection
+import spacy
+import numpy as np
+
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+
+
+class QuestionToStatement(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TAGGING,
```

Comment:
TEXT_TAGGING might not be suitable here. The Transformation should inherit TaggingOperation if it's for TEXT_TAGGING task.